src/components/crown/crownButtons/addLaneAboveButton.vue

```diff
@@ -2,10 +2,16 @@
   <crown-button
     id="lane-above-button"
     aria-label="Add lane above icon"
-    :src="laneAboveIcon"
     v-on="$listeners"
     :width="25"
-  />
+  >
+    <img
+      :src="laneAboveIcon"
+      aria-hidden="true"
+      class="lane-above-icon"
+      alt=""
```

Comment:
This is a global comment. If the alt values are going to be empty, please remove.

Reply:
Empty alt values have been removed.